int16 distance1
int16 distance2
int16 distance3
int16 distance4
int16 distance5
int16 distance6
int16 distance7